test/allowlist_for_publicAPI.json

```diff
@@ -2101,6 +2101,7 @@
     "Event",
     "EventHandlerHandle",
     "Stat",
+    "WaitCounter",
```

Comment:
Adding new entries in this file is not allowed.
Please follow the instructions from the error message to properly expose and document this class if it should be public. Or make it private properly (add a _ in front) if you don't want it to be public.

Reply:
Hm, I'm not sure I understand this policy. The rest of the types in this module are using the allow list, how exactly is this new type special?

Reply:
Also I see entries added to this file recently - https://github.com/pytorch/pytorch/commit/54b0006cb232f798281397b2261101625444c79b